File: Crypto-Trader-Website/angular/components/elements/element-group-promo/tier-promo/models/SubscriptionTier.ts
export enum SubscriptionTier {
    FREE = 'Free Tier',
    PRO = 'Pro Tier',
    ULTIMATE = 'Ultimate Tier',
}
export namespace SubscriptionTier {
    export function values(): SubscriptionTier[] {
        return [
            SubscriptionTier.FREE,
            SubscriptionTier.PRO,
            SubscriptionTier.ULTIMATE,
        ];
    }
}

File: Crypto-Trader-Library/Api-Library/Api-Models/src/main/kotlin/org/cryptotrader/api/library/entity/user/SubscriptionTier.kt
package org.cryptotrader.api.library.entity.user

enum class SubscriptionTier(val label: String, val level: Int, val intervalMs: Long) {
    FREE("Free", 1, 10000L),
    PRO("Pro", 2, 5000L),
    ELITE("Elite", 3, 1000L);

    companion object {
        fun fromLabel(label: String): SubscriptionTier? {
            return entries.find { it.label == label }
        }
    }
}
